invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/search/components.js

```diff
@@ -193,11 +193,16 @@ export const RDMRecordFacetsValues = ({
           </Label>
         </List.Content>
         {hasChildren ? (
-          <button className="iconhold"
-                  onClick={() => setisActive(!isActive)}
-                  aria-label={`${isActive ? i18next.t("hide subfacets") : i18next.t("show subfacets") }`}
-            >
-              <i className={`angle ${isActive ? "down" : "right"} icon`}></i>
+          <button
```

Comment:
why not semantic ui button?

Reply:
Because this was just refactored with prettier, didn't realise about it, I changed it,thanks!